libgit/config.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2017 Keybase Inc. All rights reserved.
+// Use of this source code is governed by a BSD
+// license that can be found in the LICENSE file.
+
+package libgit
+
+import "encoding/json"
+
+// Config is a KBFS git repo config file.
+type Config struct {
+	ID   ID
+	Name string // the original user-supplied format of the name
```

Comment:
This means we'd be able to implement rename without breaking anything right?

Reply:
This is actually so that we can have someone make a repo called `NaMe`, and call it "name" under .kbfs_git (and prevent any other casings of "name" on future repos), but still be able to display the string "NaMe" in the GUI.

We would replace this field if a rename ever happened, to preserve the format of the new name.

Reply:
I see. So repo name as we see from KBFS is always lower-cased, correct?

> We would replace this field if a rename ever happened, to preserve the format of the new name.

By replace this field, did you mean a new field in this struct, or changing what's stored in `Name`?

Reply:
> So repo name as we see from KBFS is always lower-cased, correct?

Yes.

> By replace this field, did you mean a new field in this struct, or changing what's stored in Name?

Change what's stored in `Name`.

Reply:
Understood; thanks!